src/Build/Logging/ConsoleLogger.cs

```diff
@@ -477,6 +475,30 @@ public void CustomEventHandler(object sender, CustomBuildEventArgs e)
             _consoleLogger.CustomEventHandler(sender, e);
         }
 
+        /// <summary>
+        /// Returns the minimum importance of messages logged by this logger.
+        /// </summary>
+        /// <returns>
+        /// The minimum message importance corresponding to this logger's verbosity or (MessageImportance.High - 1)
+        /// if this logger does not log messages of any importance.
+        /// </returns>
+        internal MessageImportance GetMinimumMessageImportance()
+        {
+            if (Verbosity >= BaseConsoleLogger.ImportanceToMinimumVerbosity(MessageImportance.Low, out _))
```

Comment:
Would it make sense to implement the static GetMinimumMessageImportance and ImportanceToMinimumVerbosity as extension methods on the enums instead? Should it really depend on the loggers how these two enums get mapped between themselves?

Reply:
Adding an extension method is a great idea. Same as the other comment, I think it makes sense to do when we expose the concept of minimum message importance to external loggers.